### ISSUE TITLE
will this framework work in windows 10?

### ISSUE THREAD

**Issue by RaymondDixon** [google/dopamine#21]
Just skimmed through the Document section, couldn't find any information regarding installation in Windows 10, will this framework work for windows environment?

**Comment by psc-g** [comment]
we have not tried it on windows, but i imagine it should just work with a virtualenv. see, e.g. https://programwithus.com/learn-to-code/Pip-and-virtualenv-on-Windows/

let us know if you run into any issues.

**Comment by Da-Capo** [comment]
I guess the most problem is  that you cannot install atari-py . 
Maybe you can try this https://github.com/j8lp/atari-py . it seems work for [his win10](https://github.com/openai/atari-py/issues/5#issuecomment-307647127)

**Comment by StefanUPB** [comment]
Yes, installing the dopamine package works fine on Windows with Python 3.6 (3.7 isn't supported by `tensorflow`). For `atari-py`, you can use `pip install --no-index -f https://github.com/Kojoley/atari-py/releases atari_py` as explained [on StackOverflow](https://stackoverflow.com/a/46739299/2745116).

With this the installation works. But the tests failed. Not sure yet why.

**Comment by jheffez** [comment]
Tests fail on windows. Any idea when we cam use dopamine on windows? 
Are there rainbow examples with gym (not atari)?

**Comment by mgbellemare** [comment]
I'm guessing some tests are failing because they use *nix-style paths (/tmp/...). but the code itself should work -- the only thing you may need to look out for are the paths (to checkpoints, etc.).

**Comment by jheffez** [comment]
Thanks, I'll check that. 

Are there rainbow OpenAI gym examples, that are not atari, folks can give a shot?

**Comment by jheffez** [comment]
> 
> 
> I'm guessing some tests are failing because they use *nix-style paths (/tmp/...). but the code itself should work -- the only thing you may need to look out for are the paths (to checkpoints, etc.).

I found one issue related to reading log files in Windows. The following line would just not find the log files. `utils.py, line: log_files = tf.gfile.Glob(glob)`
See [https://github.com/tensorflow/tensorflow/issues/19378](url) 
Solution is simply replace the tf.gfile glob with the standard glob:
1. import glob
2. replace above line with:
 
`Glob = os.path.join(path, '{}_[0-9]*'.format(FILE_PREFIX))

log_files = glob.glob(Glob)
` 
the upper-case Glob to prevent python from confusing withe object glob.
